thirtyinch/src/main/java/net/grandcentrix/thirtyinch/util/BackstackReader.java

```diff
@@ -17,19 +19,10 @@
      * @param fragment The fragment you want to check if its on the back stack
      * @return true, if the given Fragment is on the back stack, otherwise false (not on the back
      * stack)
-     */
-    public static boolean isInBackStack(final Fragment fragment) {
```

Comment:
With removing this we are are not able to use TI with the old support lib, right?

Reply:
That still should work but @passsy wants to get rid of this function because it's more a workaround, he has an idea already.